test/aave-v3/TestSupplyVaultBase.t.sol

```diff
@@ -20,15 +20,6 @@ contract TestSupplyVaultBase is TestSetupVaults {
         daiSupplyVault.transferRewards();
     }
 
```

Comment:
Maybe it is worth having a test checking that anyone can do it ?

Reply:
Added here: https://github.com/morpho-dao/morpho-tokenized-vaults/pull/233